doc/build.rst

```diff
@@ -70,34 +87,44 @@ Install cmake >= v3.11. E.g., the following commands install cmake v3.15.6.
    wget https://github.com/Kitware/CMake/releases/download/v3.15.6/cmake-3.15.6-Linux-x86_64.sh
    sudo bash cmake-3.15.6-Linux-x86_64.sh --skip-license --prefix=/usr/local
 
-***************************
-Building the Shared Library
-***************************
+***********************
+Building Secure XGBoost
+***********************
 
-Our goal is to build the shared library:
+Our goal is to build the shared library, along with the enclave:
 
 - On Linux the target library is ``libxgboost.so``
+- The target enclave is ``xgboost_enclave.signed``
 
 The minimal building requirement is
 
 - A recent C++ compiler supporting C++11 (g++-4.8 or higher)
 - CMake 3.11 or higher
 
-Building on Ubuntu
+Building the Targets
 ==================
 
-On Ubuntu, one builds XGBoost by running CMake:
+1. Clone the repository recursively, and check out the required branch:
 
-.. code-block:: bash
-   
-   git clone --recursive https://github.com/mc2-project/mc2-xgboost.git
-   cd mc2-xgboost
-   mkdir -p build
+   .. code-block:: bash
+
+      git clone --recursive https://github.com/mc2-project/mc2-xgboost.git
+
+2. Configure the enclave parameters in the file ``enclave/xgboost.conf``, including the amount of virtual memory available to the enclave.
```

Comment:
This is no longer relevant, right? #19 will automatically create the .conf file from the build. 

Do you think we should make the corresponding changes to the documentation in this PR, or wait until #19 is merged and then create another PR with updated documentation?

Reply:
I think we should fix this in #19. So let that PR be for now, will update it once you merge this one in.